api/.env

```diff
@@ -29,4 +30,8 @@ S3_ENDPOINT=http://minio:9000
 # Cognito
 COGNITO_AWS_REGION=eu-west-1
 COGNITO_CLIENT_ID=10h1tga4i933buv25lelalmtrn
+COGNITO_IDENTITY_POOL_ID=eu-west-1:aa0d145d-228e-40be-bb73-a9a2c83879df
 COGNITO_POOL_ID=eu-west-1_dbfEb2FuH
+
+# ION
+ION_DEFAULT_ACCESS_TOKEN='eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiI0YjNhNmQ4My01OTdlLTRjNmQtYTllYS1lMjM0NmYxZTU5ZmUiLCJpZCI6MTg3NTIsInNjb3BlcyI6WyJhc2wiLCJhc3IiLCJhc3ciLCJnYyJdLCJpYXQiOjE1NzQ0MTAwNzV9.Cj3sxjA_x--bN6VATcN4KE9jBJNMftlzPuA8hawuZkY'
```

Comment:
Is it okay to have this in the GitHub repo?

Reply:
Yes, the same token as before - it is read only. maybe we remove it in the future.